integration-tests-ex/README.md

```diff
@@ -32,21 +32,26 @@ an explanation.
 
 ### Build Druid
 
+To make the text a bit simpler, define a variable for the standard settings:
+
+```bash
+export MAVEN_IGNORE=-P skip-static-checks,skip-tests -Dmaven.javadoc.skip=true
+
 ```bash
-mvn clean package -P dist,skip-static-checks,skip-tests -Dmaven.javadoc.skip=true -T1.0C
+mvn clean package -P dist $MAVEN_IGNORE -T1.0C
```

Comment:
In the run-up to the merge, this was all replaced with a script, since it became silly to keep copy/pasting the same command lines over & over. You'll get that benefit when you rebase.